save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -127,8 +125,9 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
 
     @Suppress("EMPTY_BLOCK_STRUCTURE_ERROR", "TOO_LONG_FUNCTION", "LongMethod")
     override fun RBuilder.render() {
+        // while (true) { }
```

Comment:
what does this comment mean?

Reply:
Oops, will remove it

Reply:
Used to use it to test changes connected with `index.html` changes